cep-incremental-repodata.md

```diff
@@ -0,0 +1,289 @@
+54cafc86e0bdcc10… → 4e4344450981208d…, 10 steps
+4e4344450981208d… → a6f4223534e844e7…, 8 steps
+a6f4223534e844e7… → 99edcbd3b12a877b…, 30 steps
+99edcbd3b12a877b… → 566784db8dd57822…, 17 steps
+566784db8dd57822… → f0b32b9f6c7ec435…, 13 steps
+f0b32b9f6c7ec435… → d0f688ec3760c670…, 95 steps
+d0f688ec3760c670… → f01c8c4908735736…, 9 steps
+f01c8c4908735736… → 589ff3dbe2dec692…, 11 steps
+589ff3dbe2dec692… → 33a2c47c96a8b0c5…, 9 steps
+33a2c47c96a8b0c5… → 4f0498e69b933998…, 116 steps
+4f0498e69b933998… → deb79c8876c6d8c0…, 29 steps
+deb79c8876c6d8c0… → 848b753fcc2776b9…, 8 steps
+848b753fcc2776b9… → 377895dfffe94a3e…, 2 steps
+377895dfffe94a3e… → dbf79362ef190730…, 4 steps
+dbf79362ef190730… → 7d25389cad44036b…, 4 steps
+7d25389cad44036b… → 5ed1069fa21a7afc…, 29 steps
+5ed1069fa21a7afc… → 41cbb4960533bf5c…, 5 steps
+41cbb4960533bf5c… → 041170921a19624b…, 11 steps
+041170921a19624b… → cb26fb08146b47d2…, 6 steps
+cb26fb08146b47d2… → 37e1339f97bac34c…, 11 steps
+37e1339f97bac34c… → 6c55041cefcc67b0…, 33 steps
+6c55041cefcc67b0… → 052fa5b296656e9d…, 1 steps
+052fa5b296656e9d… → cd12fa187cc15b6e…, 1 steps
+cd12fa187cc15b6e… → b218a9437570a0f4…, 4 steps
+b218a9437570a0f4… → df124f016f7dab47…, 6 steps
+df124f016f7dab47… → ee8253efa0c0ffde…, 7 steps
+ee8253efa0c0ffde… → dea04baca7856fbe…, 3 steps
+dea04baca7856fbe… → 126550f0770b5a10…, 4 steps
+126550f0770b5a10… → 2dafc141b6629932…, 1 steps
+2dafc141b6629932… → 17a4e6b2e552a970…, 18 steps
+17a4e6b2e552a970… → 704dce0b60477958…, 1 steps
+704dce0b60477958… → b04e61a94d65d351…, 7 steps
+b04e61a94d65d351… → b692a58d5e0183d7…, 7 steps
+b692a58d5e0183d7… → e3450ce24b2c433c…, 1 steps
+e3450ce24b2c433c… → ad284fd74bf99735…, 6 steps
+ad284fd74bf99735… → 1e279bb5e0724194…, 6 steps
+1e279bb5e0724194… → 694da3d73e00d235…, 19 steps
+694da3d73e00d235… → 32e661bb5eecd5b6…, 18 steps
+32e661bb5eecd5b6… → 2cbb5a4f1a6bd09b…, 1 steps
+2cbb5a4f1a6bd09b… → 0594dbd72432f131…, 12 steps
+0594dbd72432f131… → 570e1232f56f4428…, 1 steps
+570e1232f56f4428… → adcc34a4f2f64540…, 6 steps
+adcc34a4f2f64540… → 10013ced66dc5327…, 54 steps
+10013ced66dc5327… → 5bf2dca0fb0d5e29…, 36 steps
+5bf2dca0fb0d5e29… → b0d1824a2c44ebac…, 30 steps
+b0d1824a2c44ebac… → e762a87e5bbd564e…, 19 steps
+e762a87e5bbd564e… → 31a347c12089ec9a…, 5 steps
+31a347c12089ec9a… → 2012bd26ff00f8f7…, 309 steps
+2012bd26ff00f8f7… → 3d194171754c2732…, 6 steps
+3d194171754c2732… → 23d7ca40b16a797c…, 36 steps
+23d7ca40b16a797c… → 37e1ba4f3bd8942a…, 5 steps
+37e1ba4f3bd8942a… → c9e035690378ca90…, 4 steps
+c9e035690378ca90… → 8c62fec64752f733…, 5 steps
+8c62fec64752f733… → 13604558bc1a5d65…, 1 steps
+13604558bc1a5d65… → 8d2b5e57d1c897b7…, 1 steps
+8d2b5e57d1c897b7… → 558651fbacc6e178…, 23 steps
+558651fbacc6e178… → a5661fb43183b877…, 4 steps
+a5661fb43183b877… → 300dc9b5cd8801d0…, 8 steps
+300dc9b5cd8801d0… → cd9fbab6f5af29da…, 24 steps
+cd9fbab6f5af29da… → 0aa315a3c31dd468…, 24 steps
+0aa315a3c31dd468… → bf2456c2e8ff30d5…, 31 steps
```

Comment:
This series of transitions is probably longer than it needs to be to get the point across.  I would abbreviate it like this:

```suggestion
54506a213ba69d8f… → ...
                … → bf2456c2e8ff30d5…, 31 steps
```

Reply:
Thanks. The point was supposed to be "it is pretty fast". Maybe we can just say that.